src/cdp-proxy/CDPMessageHandlers/methodsNames.ts

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for details.
+
+export const METHODS_NAMES = {
```

Comment:
I think this will be a better alternative
https://github.com/ChromeDevTools/devtools-protocol/blob/master/types/protocol-mapping.d.ts#L10